### PR TITLE
only add protocol to product urls when necessary

### PR DESCRIPTION
**Pull request by ajb**
closes #88

### CODE REVIEW

File: _layouts/product.html [hunk 1]
```diff
@@ -91,8 +91,13 @@
 				<section class="product-info">
 					<h2>{{ product.name }}</h2>
 					<h4>
-						<a href="http://{{ product.url }}"target="blank">
-							{{ product.url }}
+						{% if product.url contains "http://" or product.url contains "https://" %}
+							{% assign normalized_product_url = product.url %}
+						{% else %}
+							{% capture normalized_product_url %}http://{{product.url}}{% endcapture %}
```
Comment (benbalter) [inline _layouts/product.html:97]: This could be simplified to `{{ product.url | prepend:"http://" }}`

File: _layouts/product.html [hunk 2]
```diff
@@ -91,8 +91,13 @@
 				<section class="product-info">
 					<h2>{{ product.name }}</h2>
 					<h4>
-						<a href="http://{{ product.url }}"target="blank">
-							{{ product.url }}
+						{% if product.url contains "http://" or product.url contains "https://" %}
```
Comment (ajb) [inline _layouts/product.html:94]: I wanted a regex matcher here, or at least something to emulate "starts with", but it doesn't appear that Liquid has anything like this.

Comment (benbalter) [inline _layouts/product.html:94]: ugly, but you could match `{% product.url | split:"/" | first %}`?

Comment (ajb) [inline _layouts/product.html:94]: Yeah, but I didn't see much of a different in-practice, and I find this version to be nicely readable.

File: _layouts/product.html (hunk 1 above)
Comment (stroupaloop) [inline _layouts/product.html:97]: I'll wait to merge until this is done @ajb

Comment (ajb) [inline _layouts/product.html:97]: I think it's good to merge. Thanks though!

**Pull request merged**